api-schema/xdb.yaml

```diff
@@ -585,6 +610,33 @@ components:
         dbColumn:
           type: string
           description: the column name that can be used in the database query, see below for example   
+    LoadLocalAttributesRequest:
+      type: object
+      properties:
+        localAttributeLoadingType:
+          $ref: '#/components/schemas/LocalAttributeLoadingType'
+        partialLoadingKeys:
+          type: array
+          items:
+            type: string
+        lockingKeys:
+          type: array
+          items:
+            type: string
+    LoadLocalAttributesResponse:
+      type: object
+      properties:
+        values:
+          type: array
+          items:
+            $ref: '#/components/schemas/KeyValue'
+    LocalAttributeLoadingType:
+      type: string
+      enum:
+        - LOAD_ALL_WITHOUT_LOCKING
+        - LOAD_PARTIAL_WITHOUT_LOCKING
+        - LOAD_PARTIAL_WITH_EXCLUSIVE_LOCK
+        - LOAD_NONE
```

Comment:
I think this can just be consistent with global attributes. No "load all or load none" -- the loading/locking keys are always specified in the request. and the locking type can reuse the one from global attribute. 

One of the issues in iWF is that users are "abuse" load all too much. and I wanted to improve this in xdb. 
There is no "load all" in global attributes(hence load_none is not needed either).